src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -305,5 +306,58 @@ public async Task When_IsEnabled_Set()
 			Assert.IsTrue(textbox.IsEnabled);
 			Assert.AreEqual(contentPresenter.Foreground, foregroundColor);
 		}
+
+#if __ANDROID__
+		[TestMethod]
+		public async Task When_Text_IsWrapping_Set()
+		{
+			var textbox = new TextBox();
+
+			var initialHeight = 500;					
+			
+			textbox.Height = initialHeight;
+			
+			WindowHelper.WindowContent = textbox;
+
+			await WindowHelper.WaitForLoaded(textbox);
+
+			var actualHeigth = textbox.ActualHeight;
+
+			textbox.Text = "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremaaaaaaaaaaaaaaaaaa";
+			textbox.TextWrapping = TextWrapping.Wrap;
+			await WindowHelper.WaitForIdle();
+
+			var textBoxView = textbox.FindFirstChild<TextBoxView>();
+
+			Assert.IsTrue(textBoxView.LineCount > 1);
```

Comment:
Does this work?
```suggestion
			Assert.IsGreater(actualHeigth, textbox.ActualHeight);
```

It'd be better to use this as it will work on all platforms.